### ISSUE TITLE
question about training process

### ISSUE THREAD

**Issue by ThoamsDong** [thuml/CDAN#11]
Thanks for your impressive work! When I run your pytorch code _"Amazon to Webcam"_ in my workstation, the printed content _"iter: 00499, percision: 0.74214"_ means the test percision is only 0.74214, which is far from the percision you memtioned in _"Conditional Adversarial Domain Adaptation"_. After training for a long time, the percision cannot improve. the Could you tell me how to train and test it? 
![1](https://user-images.githubusercontent.com/34698969/55852042-f1fe4500-5b8d-11e9-8bed-b79e94a2367a.PNG)

**Comment by caozhangjie** [comment]
Which PyTorch version are you using?

**Comment by ThoamsDong** [comment]
![1](https://user-images.githubusercontent.com/34698969/55855560-722aa780-5b9a-11e9-9197-9d839b926f3d.PNG)

**Comment by caozhangjie** [comment]
You may try 0.4.0 as we stated.

**Comment by ThoamsDong** [comment]
Thanks for your reply, and I will try it.

**Comment by ThoamsDong** [comment]
I am so sorry to tell you that it gets the same result. And I do not know how to deal with it?
![1](https://user-images.githubusercontent.com/34698969/55886477-67920180-5bde-11e9-9bfa-caf26a7ed162.PNG)

**Comment by binity** [comment]
Thank you for your excellent job, but there is one question confused me a lot, could you explain why there is only one loss in the training progess, I think it is commen two have two loss function,a lossG and a lossD during the adversarial learning.BTW,I'm reading  the PyTorch code. I'll be appreciate if you can explain it to me!

**Comment by sy565612345** [comment]
> Thank you for your excellent job, but there is one question confused me a lot, could you explain why there is only one loss in the training progess, I think it is commen two have two loss function,a lossG and a lossD during the adversarial learning.BTW,I'm reading the PyTorch code. I'll be appreciate if you can explain it to me!
Hi,
We combine the lossG('classifier_loss' in the code) and lossD('transfer loss in the code') in the 'total_loss'.
I hope this can help you.

**Comment by sy565612345** [comment]
> Thank you for your excellent job, but there is one question confused me a lot, could you explain why there is only one loss in the training progess, I think it is commen two have two loss function,a lossG and a lossD during the adversarial learning.BTW,I'm reading the PyTorch code. I'll be appreciate if you can explain it to me!

Hi,
We combine the lossG('classifier_loss' in the code) and lossD('transfer loss in the code') in the 'total_loss'.
I hope this can help you.

**Comment by curryandone** [comment]
> I am so sorry to tell you that it gets the same result. And I do not know how to deal with it?
> ![1](https://user-images.githubusercontent.com/34698969/55886477-67920180-5bde-11e9-9bfa-caf26a7ed162.PNG)

Has that problem been solved?I meet the same problem.

**Comment by akshaymehra24** [comment]
> Thanks for your impressive work! When I run your pytorch code _"Amazon to Webcam"_ in my workstation, the printed content _"iter: 00499, percision: 0.74214"_ means the test percision is only 0.74214, which is far from the percision you memtioned in _"Conditional Adversarial Domain Adaptation"_. After training for a long time, the percision cannot improve. the Could you tell me how to train and test it?
> ![1](https://user-images.githubusercontent.com/34698969/55852042-f1fe4500-5b8d-11e9-8bed-b79e94a2367a.PNG)

Were you able to fix this? Im trying to use their code but can't replicate the numbers reported in the paper.
